### PR TITLE
Event index route

### PR DESCRIPTION
**Pull request by gharmonjr**
Creates the index page for the events route. Uses the IndexEventOperation.

### CODE REVIEW

File: spec/examples.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-example_id                                               | status | run_time        |
--------------------------------------------------------- | ------ | --------------- |
-./spec/operations/index_event_operation_spec.rb[1:1:1:1] | passed | 0.19676 seconds |
+example_id                                               | status | run_time      |
```
Comment (BrianLees) [inline spec/examples.txt]: Do we want to remove this from source control?

**Review by BrianLees — APPROVED** [review]
Couple minor things, but not deal breaking.

File: spec/examples.txt (hunk 1 above)
Comment (gharmonjr) [reply spec/examples.txt]: yes, good call.

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -4,7 +4,12 @@ class EventsController < ApplicationController
   # GET /events
   # GET /events.json
   def index
-    @events = Event.all
+    @events = IndexEventOperation.new(user_id).results
+
+    respond_to do |format|
```
[the remark this answers is not in the document]
Comment (gharmonjr) [reply app/controllers/events_controller.rb]: similar to the create and update methods, this enables a .json on the established route. By default we'll get the html route

[the remark this answers is not in the document]
Comment (gharmonjr) [reply app/controllers/events_controller.rb]: image

**Pull request merged**